integration-tests/tests.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python3
+
+import subprocess, argparse, shlex, sys, re
+
+def run_tests(behaviour, args):
+    if behaviour in {'put', 'get'}:
+        try:
+            print(f'Running bobp -b {str(behaviour)} {args.rstrip()}')
+            p = subprocess.check_output(shlex.split(f'./bobp -b {behaviour} {args.rstrip()}'))
+            print(str(p))
+            if f'{behaviour} errors:' in str(p) or f'panicked' in str(p):
+                sys.exit(f'{behaviour} test failed, see output')
+        except subprocess.CalledProcessError as e:
+            sys.exit(str(e.stderr))
+    elif behaviour == 'exist':
+        try:
+            print(f'Running bobp -b {str(behaviour)} {args.rstrip()}')
+            p = subprocess.check_output(shlex.split(f'./bobp -b {behaviour} {args.rstrip()}'))
+            print(str(p))
```

Comment:
These 3 lines above are common for both branches. I suggest to make them common by moving them out of `if..else` statement